db/migrate/20221129184639_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Please use `author_id` for posts instead of `user_id` to match the ERD diagram.
  - You can use this code ` add_references :author, null: false, foreign_key: {to_table: 'users'}, index: true`